### PR TITLE
Pr drone can support flight test patch

### PR DESCRIPTION
**Pull request by Huibean**
patch for first flight test
1.current value convert fix
2.add motor kv param

### CODE REVIEW

File: Src/DroneCAN/DroneCAN.c [hunk 1]
```diff
@@ -263,6 +266,14 @@ static void handle_param_GetSet(CanardInstance* ins, CanardRxTransfer* transfer)
 		eepromBuffer[p->eeprom_index] = *(uint8_t *)p->ptr;
 	    }
             break;
+	case T_UINT16:
+	    *(uint16_t *)p->ptr = req.value.integer_value;
+	    if (p->eeprom_index != 0) {
+            if (p->eeprom_index == 26) {
+                eepromBuffer[26] = (uint8_t)((*(uint16_t *)p->ptr - 20) / 40);
```
Comment (tridge) [inline Src/DroneCAN/DroneCAN.c]: it would be good to have a comment explaining this :-)

**Review by tridge — CHANGES_REQUESTED** [review]

File: Src/DroneCAN/DroneCAN.c (hunk 1 above)
Comment (Huibean) [reply Src/DroneCAN/DroneCAN.c]: change it to readable define EEPROM_MOTOR_KV_INDEX, btw, the code is tab and space mix a little bit mess to correct the format

**Comment by Huibean** [comment]
3. add tune edit feature
work with dronecan gui tool in this PR https://github.com/dronecan/gui_tool/pull/62

**Comment by Huibean** [comment]
4.add HSE support and VIMDRONES_NANO_CAN_L431 target

**Comment by tridge** [comment]
i've rebased this

**Review by tridge — APPROVED** [review]

**Pull request merged**